.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,30 @@
+name: 'Bug Report（bug 提交）'
+about: 'Report a software bug on this repo.（提交本仓库的 Bug）'
+title: ''
+labels:
+- 'bug'
+assignees: []
```

```suggestion
labels:
  - bug
assignees:
  - TechQuery
```
1. YAML 的列表和键值对相当于编程语言当中的数组和对象，不可能在同一级同时存在数组和字符串索引，所以两级一定有缩进。

2. YAML 没有特殊符号的字符串可以不用加引号。

3. issue 的指派人可设为仓库排名靠前的贡献者。